### PR TITLE
3rd party manifests in pipeline with internal manifests

### PR DESCRIPTION
**Pull request by drpebcak**

### CODE REVIEW

**Comment by drpebcak** [comment]
Failure is unrelated to my change.

File: .drone.yml [hunk 1]
```diff
@@ -364,6 +364,26 @@ steps:
...
+      from_secret: docker_username
+    DOCKER_PASSWORD:
+      from_secret: docker_password
+  privileged: true
+  volumes:
+  - name: docker
+    path: /var/run/docker.sock
+  when:
+    branch:
+    - master
```
Comment (cloudnautique) [inline .drone.yml]: This should also only be done on tag.

Comment (cloudnautique) [reply .drone.yml]: @alena1108 do we need these on RCs or only on final builds?

Comment (drpebcak) [reply .drone.yml]: I would assume we would want it for RCs and alphas, because otherwise they might fail to bring up those services.

Comment (drpebcak) [reply .drone.yml]: So pretty much any tag?

**Pull request closed without merging**